scst-scan/install-prisma-integration.hbs.md

```diff
@@ -0,0 +1,566 @@
+ * VERSION is your Prisma Scanner version. For example, 0.1.0-beta.4
+ * TARGET-REPOSITORY is your target repository, a folder/repository on MY-REGISTRY that serves as the location for the installation files for Prisma Scanner.
+
+5. Install the Carvel tool imgpkg CLI.
+
+6. Relocate the images with the imgpkg CLI by running:
+
+```
+imgpkg copy -b projects.registry.vmware.com/tap-scanners-package/prisma-repo-scanning-bundle:${VERSION} --to-repo ${INSTALL_REGISTRY_HOSTNAME}/${INSTALL_REPO}/prisma-repo-scanning-bundle
+```
+
+## Add the Prisma Scanner package repository
+
+Tanzu CLI packages are available on repositories. Adding the Prisma Scanning package repository makes Prisma Scanning bundle and its packages available for installation.
+
+> **Note**
+> Relocate images to a registry is strongly recommended but not required for installation. For simplicity, this section assumes you have relocated images to a registry. Refer to that section to fill in the stubbed out variables.
+
+
+It's recommended to install the Prisma Scanner objects in the existing tap-install namespace to keep the Prisma Scanner grouped logically with the other TAP components.
+
+1. Add the Prisma Scanner package repository to the cluster by running:
+
+```
+tanzu package repository add prisma-scanner-repository \
+  --url ${INSTALL_REGISTRY_HOSTNAME}/${INSTALL_REPO}/prisma-repo-scanning-bundle:$VERSION \
+  --namespace tap-install
+```
+
+2. Get the status of the Prisma Scanner package repository, and ensure the status updates to Reconcile succeeded by running:
+
+```
+tanzu package repository get prisma-scanning-repository --namespace tap-install
+```
+
+For example:
+
+```
+$ tanzu package repository get prisma-scanning-repository --namespace tap-install
+- Retrieving repository prisma-scanner-repository...
+NAME:          prisma-scanner-repository
+VERSION:       71091125
+REPOSITORY:    index.docker.io/tapsme/prisma-repo-scanning-bundle
+TAG:           0.1.0-beta.4
+STATUS:        Reconcile succeeded
+REASON:
+```
+
+3. List the available packages by running:
+
+```
+tanzu package available list --namespace tap-install
+```
+
+For example:
+
+```
+$ tanzu package available list --namespace tap-install
+/ Retrieving available packages...
+  NAME                                                 DISPLAY-NAME                                                              SHORT-DESCRIPTION
+  prisma.scanning.apps.tanzu.vmware.com                Prisma for Supply Chain Security Tools - Scan                             Default scan templates using Prisma                                              
+```
+
+## Prequisites for Prisma Scanner (Beta)
+
+This document describes prerequisites for installing Supply Chain Security Tools - Scan (Prisma) from the VMware package repository.
+
+### Prepare the Prisma Scanner configuration
+
+To prepare the Prisma configuration before you install any scanners:
+
+1. Obtain a Prisma Token from Prisma. 
```

Comment:
Is there a link for this?

Reply:
Not necessary, link could change in the future, and the user would most likely know how to get it already.